test/es6/bug_issue_3076.js

```diff
@@ -0,0 +1,328 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function f0 (a, b) {
```

Comment:
not sure why this many expression needed. Does this repro in the forcedeferparse?